inception-project/inception
# #2115 - Better info when running in unsupported Java

**What's in the PR**
- Reduce Java target in boot module to Java 1.8
- Display message / open window when running on a version lower than Java 11

**How to test manually**
* Start with different Java versions older and newer than Java 11 (also Java 11)
* Run with or without GUI on the older versions to see if the message gets shown properly

**Automatic testing**

**Documentation**
